respa_admin/static_src/styles/base.scss

```diff
@@ -29,4 +29,5 @@ $icon-font-path: "~bootstrap-sass/assets/fonts/bootstrap/";
 @import "./page-office";
 @import "./user-matrix";
 @import "./page-resource";
+@import "./page-user-management-list.scss";
```

Comment:
I'm interested - why is this import listed with file extension while the others are not?

Reply:
No particular reason, I was unaware of the magic handling the imports. Will change.